common/models/logbook.js

```diff
@@ -73,28 +73,23 @@ module.exports = function (Logbook) {
   /**
      * Find Logbook model instance by name
      * @param {string} name Name of the Logbook
-     * @param {string} filters Filter rison object, keys: textSearch, showBotMessages, showUserMessages, showImages, skip, limit, sortField
+     * @param {string} filters Filter json object, keys: textSearch, showBotMessages, showUserMessages, showImages, skip, limit, sortField
      * @returns {Logbook} Logbook model instance
      */
 
   Logbook.findByName = async function (name, filters) {
     if (logbookEnabled) {
       try {
         const accessToken = await login(username, password);
-        const decodedFilters = filters
-          ? rison.decode_object(filters)
-          : undefined;
         console.log("Fetching logbook", { name, filters });
         const res = await superagent
           .get(
             baseUrl +
-                            `/Logbooks/${name}?filter=${JSON.stringify(
-                              decodedFilters
-                            )}`
+                            `/Logbooks/${name}?filter=${JSON.stringify(filters)}`
```

Comment:
JSON.stringify should be removed here, otherwise query won't work.

Reply:
Should I assume that the argument filter to this function is already a json string?
If so, should we change the comment and stating clear that fileter is the stringified version of the filter json object?

Reply:
Done.
We should think about implementing some tests for the logbook functions.